workflows/seqqc.nf

```diff
@@ -176,7 +176,32 @@ workflow SEQQC {
 
 workflow.onComplete {
     if (params.email || params.email_on_fail) {
-        NfcoreTemplate.email(workflow, params, summary_params, projectDir, log, multiqc_report)
+        def email_params = NfcoreTemplate.get_email_params(workflow, params, summary_params, projectDir, log, multiqc_report)
```

Comment:
ok so still using nf-core email templating, but not using nf-core send mail stuff

Reply:
yes